src/PKSim.Presentation/Presenters/Protocols/ProtocolChartData.cs

```diff
@@ -73,60 +69,45 @@ public ProtocolChartData(ICache<Compound, IReadOnlyList<SchemaItemDTO>> allSchem
          DataTable = new DataTable("Protocol");
          TimeDimension = timeDimension;
          TimeUnit = timeUnit;
-         DataTable.Columns.Add(new DataColumn(GROUPING_NAME, typeof (string)));
-         DataTable.Columns.Add(new DataColumn(TIME, typeof (double)));
-         DataTable.Columns.Add(new DataColumn(DOSE, typeof (double)));
-         DataTable.Columns.Add(new DataColumn(UNIT, typeof (Unit)));
-         DataTable.Columns.Add(new DataColumn(SCHEMA_ITEM, typeof(SchemaItemDTO)));
+         DataTable.AddColumn(GROUPING_NAME);
```

Comment:
`AddColumn `without type qualifier assumes _Type=string_? Still would prefer `AddColumn<string>` instead

Reply:
It's used like that everywhere and there is no confusion possible when the column name are clearly defined (such as GROUPING_NAME). Nothing to change in my opinion